lint.js

```
@@ -1,3 +1,9 @@
 const add = function(a, b) {
     return a + b;
 }
+const add2 = function(a, b) {
+    return a + b;
+}
+const equal = function(a, b) {
```

ESLint

[error] 'equal' is assigned a value but never used.

(no-unused-vars)